common/src/main/java/com/wynntils/screens/maps/widgets/PoiManagerWidget.java

```diff
@@ -30,15 +30,15 @@
 import net.minecraft.network.chat.Component;
 
 public class PoiManagerWidget extends AbstractWidget {
-    private CustomPoi poi;
-    private Button editButton;
-    private Button deleteButton;
-    private Button upButton;
-    private Button downButton;
-    private int row;
+    private final CustomPoi poi;
+    private final Button editButton;
+    private final Button deleteButton;
+    private final Button upButton;
+    private final Button downButton;
+    private final int row;
     private CustomColor color;
```

Comment:
Quite unfortunate ordering, can you move color to be last? (What is even setting color...)